portfolio/src/main/webapp/about.css

```diff
@@ -0,0 +1,42 @@
+.font {
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.link {
+    color: white;
+} :hover {
+    color: black;
+}
+
+.abt-title {
+  padding-top: 30px;
+  font-weight: 600;
+  font-size: 48px;
+}
+
+.abt-content {
+  margin-left: 20%;
+  margin-right: 20%;
+  font-size: 20px;
+  text-align: center;
+} div {
```

Comment:
Same here, pretty sure this will apply to all `div`s at the moment (feel free to verify in the styles tab of the developer tools). If you want all `div`s that have `.abt-content` as an ancestor, the selector should be `.abt-content div` (or `.abt-content > div` for only direct children).

Reply:
fixed!